playbook/app/pb_kits/playbook/pb_collapsible/collapsible.html.erb

```diff
@@ -2,6 +2,7 @@
     aria: object.aria,
     class: object.classname,
     data: object.data,
-    id: object.id) do %>
+    id: object.id,
+    role: "region") do %>
```

Comment:
Adds `role` to the wrapper. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

Reply:
++